packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,214 @@
+---
+title: OIN submission requirements
+meta:
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary for the submission process using the OIN Manager.
+
+**What you need**
+
+* a built and tested OIN integration that is ready for Okta verification
+
+---
+
+## Overview
+
+Prior to using the [OIN Manager](https://oinmanager.okta.com) to submit your OIN integration, you need to review the following guidelines to prepare for the artifacts requested during the submission process:
+
+* [Logo guidelines](#logo-guidelines)
+* [App description guidelines](#app-description-guidelines)
+* [Use case guidelines](#use-case-guidelines)
+* [Test account guidelines](#test-account-guidelines)
+* [Customer configuration document guidelines](#customer-configuration-document-guidelines)
+
+Before you submit your integration, ensure that your integration doesn't use unsupported OIN Okta features. See [OIN limitations](#oin-limitations).
+
+## Logo guidelines
+
+The app logo that you submit to the OIN Manager in the **General Settings** > **App icon** field must conform to the following guidelines:
+
+* The logo file size must be less than one MB.
+* The recommended logo dimension is square (1:1 aspect ratio).
+
+   > **Note:** OIN app icons are placed in a 200 x 200 pixels grid.
+
+* The recommended logo contains only a logomark and not a logotype (such as a company icon and not the company name).
+
+   > **Note:** The company name is already listed in plain text in the OIN catalog, so adding a company name image is redundant.
+
+* Don't use logos with both an icon and a wordmark.
```

Comment:
Still not sure what the difference is between this one and the one above

Reply:
right, i'll collapse the 2 points.

Reply:
actually, i've updated this to 2 points:
```
* Don't use logos with both an icon and a wordmark.
* Don't use just a wordmark logo (such as the company name).
```
Let me know if this is ok?

Reply:
Feel like I'm nit picking at this point, but isn't it basically:
* Submit an icon rather than a wordmark / logo which includes the company or product name

Reply:
No worries about the nit-picking ;-)
I just separated the two points because in the original Logo guideline handout, these points were emphasized so I thought the OIN Ops team wanted to do the same in this requirements doc.
I'll use your suggestion.